static/templates/head_part.html

```diff
@@ -5,6 +5,7 @@
 <script type="text/javascript" src="{{root}}/skin/jquery-ui/jquery-ui.min.js"></script>
 <script>
   var jk = jQuery.noConflict();
+  // block external links
```

Comment:
This file is included only if we have a taskbar. As I already said you should not include the catpure function inside the taskbar.

This line will be probably removed in the next version of this PR anyway, but I just want to mention that I would prefer a comment like `//<BLOCK_EXTERNAL> <-- block external function will be inserted here`.
If you read this file without the context of this PR, it seems it is just a comment out of context and someone will probably remove it next time he changes this file.